account/views.py

```diff
@@ -146,6 +146,48 @@ def get(self, request):
             return Response({'status': 'success', 'Response': {'email':email, 'first_name':first_name, 'last_name':last_name, 'company':company, 'manager':manager}}, 
                                 status=status.HTTP_200_OK)
 
+        except Exception as e:
+            return Response({'status': False, 'message': str(e)},
+                            status=status.HTTP_400_BAD_REQUEST)
+
+employee_id = openapi.Parameter('id', in_=openapi.IN_QUERY,
+                         type=openapi.TYPE_STRING)   
+
+class GetEmployeeInfo(APIView):
+    @swagger_auto_schema(
+        manual_parameters=[employee_id],
+    )
+    def get(self, request):
+        try:
+            user = request.user
+            account = user.account_set.first()
+            manager_id = account.id
+
+            employee_id = request.GET.get('id')
+            if employee_id != None:
+                employee_accounts = Account.objects.filter(id=employee_id)
+                if len(employee_accounts) == 0:
+                    return Response({'status': False, 'message': "Invalid employee id"},
+                                    status=status.HTTP_403_FORBIDDEN)
+                employee_account = employee_accounts[0]
+
+                if employee_account.manager == None or employee_account.manager.id != manager_id:
+                    return Response({'status': False, 'message': "Data is forbidden"},
+                                    status=status.HTTP_403_FORBIDDEN)
+
+                return Response({'status': 'success', 'Response': {'email': employee_account.user.email, 'first_name': employee_account.user.first_name, 'last_name': employee_account.user.last_name, 'company': employee_account.company.name}}, 
```

Comment:
As I said earlier in the prev class, you can return the whole object's data with a single serializer. Check it out. You can still compose a dictionary and return it like this, but the problem is it doesn't look that concise though. It's up to you to decide. I know finishing the task asap is the main priority so I don't want to stress this too much.